mrp_workcenter_hierarchical/models/mrp_workcenter.py

```diff
@@ -0,0 +1,61 @@
+# 2016 Akretion (http://www.akretion.com)
+# David BEAL <david.beal@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class MrpWorkcenter(models.Model):
+    _inherit = "mrp.workcenter"
+    _parent_name = "parent_id"
+    _parent_store = True
+
+    parent_id = fields.Many2one(
+        comodel_name="mrp.workcenter", string="Parent", index=True, ondelete="cascade"
```

Comment:
I am not sure about the` ondelete="cascade"` here.
Since a bit hard to try to delete it if parent is deleted... I think it would be best to leave the workcenter alone, with no parent and the user will delete it if needed what do you think?

Reply:
@bealdav ?

Reply:
Resolved